packages/react-server-cli/src/commands/start.js

```diff
@@ -23,7 +23,7 @@ export default function start(options){
 
 	const {
 		port,
-		host,
+		bind: bindIp = "0.0.0.0",
```

Comment:
What is this I don't even. :squint:

Cool inline default syntax.  Shouldn't defaults go in `defaultOptions.js`, though?